clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -102,7 +102,11 @@ func (p *podStartupLatencyMeasurement) Execute(config *measurement.Config) ([]me
 		}
 		return nil, p.start(config.ClusterFramework.GetClientSets().GetClient())
 	case "gather":
-		return p.gather(config.ClusterFramework.GetClientSets().GetClient(), config.Identifier)
+		schedulerName, err := util.GetString(config.Params, "schedulerName")
```

Comment:
Tests failed, because GetString expects parameter to always be there. You can replace it by "GetStringOrDefault" and provide corev1.DefaultSchedulerName as default

Reply:
Thanks, @marseel for the suggestion, I have made the changes, please review!